integreat_cms/cms/views/organizations/organization_list_view.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from django.conf import settings
+from django.contrib import messages
+from django.core.paginator import Paginator
+from django.shortcuts import render, redirect
+from django.utils.decorators import method_decorator
+from django.utils.translation import ugettext as _
+from django.views.generic import TemplateView
+from django.views import View
+from django.http import HttpResponse
+from django.views.generic.base import ContextMixin
+
+
+logger = logging.getLogger(__name__)
+
+
+class OrganizationListView(TemplateView):
+    template_name = "organizations/organization_list.html"
+    extra_context = {
+        "current_menu_item": "organizations",
+    }
+
+    def get(self, request, *args, **kwargs):
+        region = request.region
+        if not request.user.has_perm("cms.change_organization"):
+            messages.warning(
+                request,
+                ("You don't have the permission to edit or create organizations."),
+            )
+
+        organizations = region.organizations.all()
+        chunk_size = int(request.GET.get("size", settings.PER_PAGE))
+        paginator = Paginator(
+            organizations.order_by("created_date"),
```

Comment:
```suggestion
            organizations
```
I think the default ordering is sufficient here.

Reply:
Does not make much of a difference I guess, but without calling order it might be also be a bit faster.

Reply:
Yeah, performance difference is probably not noticeable to humans :wink:
It comes down to how we want to order the table (at the moment, it's by the `slug`) - if we want to change this, I would directly change it in the [model](https://github.com/digitalfabrik/integreat-cms/blob/be78a63130bac631f95a6c020e0764d916488759/integreat_cms/cms/models/users/organization.py#L74)... no me, either `last_updated` or `name` make the most sense, but we can also wait until the service team complains :grin:

Reply:
That seems like a valid strategy as well :D